quic/s2n-quic-platform/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-quic-platform"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
Since core got a minor bump, i think we also want to also do a minor bump for this crate.

Looking at the changes, maybe this could be a patch change, but then we are trying to figure out how significant the change was. I say if the dependency crate saw a minor change then this crate also sees a minor change.

Reply:
Okay, will make this a minor bump